pkg/operator/sync.go

```diff
@@ -329,7 +329,7 @@ func (optr *Operator) syncRequiredMachineConfigPools(config renderConfig) error
 		return err
 	}
 	var lastErr error
-	if err := wait.Poll(time.Second, 5*time.Minute, func() (bool, error) {
+	if err := wait.Poll(time.Second, 10*time.Minute, func() (bool, error) {
```

Comment:
I feel like these timeouts may still be too short.  But we can revisit that later.

Reply:
well, the installer waits up to 30 mins anyway, we could bump up to that if needed I guess